src/components/nodes/node-stanford-person.tsx

```diff
@@ -20,104 +21,158 @@ export const NodeStanfordPerson = ({node, ...props}: PersonNodeProps) => {
         <NextSeo
           openGraph={{profile: {firstName: node.su_person_first_name, lastName: node.su_person_last_name}}}
         />
-
-        <div className="su-flex su-no-wrap su-mb-[50px]">
-
-          {node?.su_person_photo?.field_media_image &&
-              <div className="su-mr-[20px]">
-                  <div className="su-rounded-full su-w-[220px] su-h-[220px] su-overflow-hidden">
-                      <DrupalImage
-
-                          src={node.su_person_photo.field_media_image.uri.url}
-                          alt={node.su_person_photo.field_media_image.resourceIdObjMeta.alt}
-                          height={node.su_person_photo.field_media_image.resourceIdObjMeta.height}
-                          width={node.su_person_photo.field_media_image.resourceIdObjMeta.width}
-                      />
-                  </div>
-              </div>
-          }
+        <div className="sm:su-flex su-no-wrap su-rs-mb-5 su-mt-50">
+          <Conditional showWhen={node.su_person_photo.field_media_image}>
+            <div className="su-rs-mr-4">
+                <div className="su-rounded-full su-w-[220px] su-h-[220px] su-overflow-hidden">
+                    <DrupalImage
+                        src={node.su_person_photo.field_media_image.image_style_uri.medium_square}
+                        alt={node.su_person_photo.field_media_image.resourceIdObjMeta.alt}
+                        height={node.su_person_photo.field_media_image.resourceIdObjMeta.height}
+                        width={node.su_person_photo.field_media_image.resourceIdObjMeta.width}
+                    />
+                </div>
+            </div>
+          </Conditional>
 
           <div>
-            {node.su_person_short_title}
+            <Conditional showWhen={node.su_person_short_title}>
+              <div className="su-type-0 su-leading">{node.su_person_short_title}</div>
+            </Conditional>
             <h1>{node.title}</h1>
```

Comment:
I know this will change it from the core product, but the title should be placed into the `<MainContentLayout>` so that it displays within the black bar at the top.